cmake/std/trilinosprhelpers/TrilinosPRConfigurationInstallation.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- mode: python; py-indent-offset: 4; py-continuation-offset: 4 -*-
+"""
+Custom PR Executor for Installation testing
+"""
+
+import os
+import subprocess
+
+#from . import setenvironment
+#from . import sysinfo
+#from . import jenkinsenv
+
+from . import TrilinosPRConfigurationBase
+
+
+
+class TrilinosPRConfigurationInstallation(TrilinosPRConfigurationBase):
+    """
+    Implements Standard mode Trilinos Pull Request Driver
```

Comment:
Installation mode?

Reply:
BTW, does this differ from the `Standard` one, or not yet?

Reply:
Installation Testing... it will work differently from a standard test.  It's stubbed in for now.